services/openshift-service-impl/src/main/java/io/openshift/appdev/missioncontrol/service/openshift/impl/fabric8/openshift/client/Fabric8OpenShiftServiceImpl.java

```diff
@@ -310,6 +310,17 @@ private void configureProject(final OpenShiftProject project, final InputStream
                             ((OpenShiftProjectImpl) project).addResource(resource);
                         });
             }
+
+            // Add Admin role to the jenkins serviceaccount
+            log.info("Adding role admin to jenkins serviceaccount for project '" + project.getName() + "'");
```

Comment:
log.info is used to log all the info within this method. So, do you want to change it also for the others log. ?